bundles/org.openhab.io.openhabcloud/src/main/resources/OH-INF/config/config.xml

```diff
@@ -16,13 +16,16 @@
 		</parameter>
 		<parameter name="expose" type="text" required="false" multiple="true">
 			<label>Items to Expose</label>
-			<description>List of items that are made accessible to IFTTT and similar services.</description>
+			<description>List of items that are made accessible to IFTTT and similar services. Only supported on private cloud
```

Comment:
Should this be updated also to not mention IFTTT, but perhaps Amazon Alexa and Google Home applications? Also, i18n properties should be regenerated.

Reply:
It specifically only used for IFTTT and for anything else that wants to use the limited cloud api but NOT for Alexa and google, and it only works if running a private cloud.  We have disabled this support on the main myopenhab.org service through a runtime option.  This is also why it makes sense  to put behind the advanced flag, only users who can standup their own cloud service would be interested in it.

Reply:
>Also, i18n properties should be regenerated

Ahh, did not realize there were translations, I updated the main property file, is there anything else i need to do?

Reply:
No, that's exactly what was needed. :)